### PR TITLE
Bump version and docs

### PR DESCRIPTION
**Pull request by andrewshkovskii**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 dist
 build
 MANIFEST
+.idea/*
```
Comment (dmitry-mukhin) [inline .gitignore]: :(

File: AUTHORS [hunk 2]
```diff
@@ -7,3 +7,4 @@ Contributors:
 - Marek Malek
 - homm
 - Alexander Zhukov
+- Andrewshkovskii
```
Comment (dmitry-mukhin) [inline AUTHORS]: :(

**Comment by andrewshkovskii** [comment]
@dmitry-mukhin Better now?

File: AUTHORS [hunk 3]
```diff
@@ -5,6 +5,8 @@ Contributors:
 - @omansion
 - Mike Harris
 - Marek Malek
-- homm
 - Alexander Zhukov
-- Andrewshkovskii
\ No newline at end of file
+- Uploadcare team:
```
Comment (dmitry-mukhin) [inline AUTHORS]: :)

**Pull request merged**